src/main/java/org/stellar/sdk/SorobanServer.java

```diff
@@ -560,6 +544,52 @@ public SendTransactionResponse sendTransaction(Transaction transaction) {
         "sendTransaction", params, new TypeToken<SorobanRpcResponse<SendTransactionResponse>>() {});
   }
 
+  public GetSacBalanceResponse getSacBalance(String contractId, Asset asset, Network network)
```

Comment:
I am not sure if creating the `GetSacBalanceRequest` class would be better because it is translated to `GetLedgerEntriesRequest`.

Reply:
I think it's fine just the way it is now, no need to add anything.